Frameworks/CoreText/CTFrame.mm

```diff
@@ -108,13 +108,14 @@ void CTFrameDraw(CTFrameRef frame, CGContextRef ctx) {
 
         CGPoint relativePosition = frame->_lineOrigins[i];
 
-        for (size_t j = 0; j < [line->_runs count]; ++j) {
-            _CTRun* curRun = [line->_runs objectAtIndex:j];
+        CFIndex len = CFArrayGetCount(line->_runs);
+        for (size_t j = 0; j < len; ++j) {
+            __CTRun* curRun = const_cast<__CTRun*>(static_cast<CTRunRef>(CFArrayGetValueAtIndex(line->_runs, j)));
```

Comment:
&curRun->_dwriteGlyphRun 
is holding it via a pointer which isn't non-const.
I can downcast it here `(GlyphRunData{ &curRun->_dwriteGlyphRun` but i'll just be safe and leave it very similar to how it was before.